src/app/main/component/user/components/profile/users-friends/friend-dashboard/friend-item/friend-item.component.ts

```diff
@@ -61,6 +61,11 @@ export class FriendItemComponent implements OnInit {
     this.router.navigate([this.friend.name, this.friend.id], { relativeTo: this.route, queryParams: { tab: 'All firends', index: 3 } });
   }
 
+  public cutText(): string {
+    const maxTextLength = 20;
```

Comment:
It would be better to create a variable for 20. And also we have already pipe for this functionality `maxImageName`. But maybe we should rename it if you use it)